style.css

```diff
@@ -0,0 +1,269 @@
+	color:#f7f7f7;
+}
+.hello-section h2 {
+	font-size: 3rem;
+	color:#f7f7f7;
+}
+/*ABOUT SECTION*/
+.about-section {
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+	width: 100%;
+	padding: 0 10px;
+}
+.about-section h2 {
+	font-size: 2rem;
+	margin-left: 15px;
+	color:#f7f7f7;
+}
+.about-section h3 {
+	font-size: 1.5rem;
+	padding: 0;
+	margin-top: 0;
+	margin-left: 15px;
+	color: #f7f7f7;
+	width:100%;
+	
+}
+/*PORTFOLIO SECTION*/
+.portfolio-section {
+	display: flex;
+	flex-wrap: nowrap;
+	flex-direction: column;
+	gap: 10px;
+}
+.portfolio-description {
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+	width: 100%;
+	background-color: #000000;
+	padding: 0 10px;
+}
+.portfolio-description h2 {
+	font-size: 2rem;
+	margin-left: 15px;
+	color: #D9D9D9;
+}
+.portfolio-description h3 {
+	font-size: 1.5rem;
+	padding: 0;
+	margin-top: 0;
+	margin-left: 15px;
+	color: #D9D9D9;
+}
+.card {
+	display: flex;
+	flex-wrap: wrap;
+	width: 100%
+	flex-direction: column;
+	max-width: 1200px;
+	margin-left: auto;
+	margin-right: auto;
+	border: 1px solid black;
+	border-radius: 20px;
+	
+}
+.card h3 {
+	font-size: 2rem;
+	margin-left: 15px;
+	margin-bottom: 10px;
+	padding: 0 30px;
+	margin-left: 10px;
+	flex-direction: row;
+	padding-top: 0;
+	margin-top: 15px;
+	color:#f7f7f7;
+	
+}
+.card-img-text {
+	display: flex;
+	flex-wrap: nowrap;
+	margin-left: auto;
+	margin-right: auto;
+	flex-direction: row;
+}
+.card-img-text img {
+	width: 400px;
+	aspect-ratio: auto;
+	justify-content: flex-start;
+	margin-left: 10px;
+	margin-bottom: 10px;
+	border-radius: 10px;
+	border: 1px solid #d9d9d9;
+	
+}
+.card p {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	padding: 0 40px;
+	font-size: 1.5rem;
+	color:#f7f7f7;
+}
+
+/*CONTACT SECTION*/
+.contact-section {
+	margin-top: 10px;
+	display: flex;
+	flex-wrap: wrap;
+	flex-direction: column;
+	width: 100%;
+	justify-content: center;
+}
+
+.contact-section h2 {
+	font-size: 1.75rem;
+	margin-left: 15px;
+	margin-bottom:0;
+	color:#f7f7f7;
+}
+.contact-section h3 {
+	font-size: 1.5rem;
+	margin-left: 15px;
+	margin-bottom:3px;
+	color:#f7f7f7;
+	margin-top:10px;
+	margin-bottom:5px;
+}
+.logo {
+	margin-left: 15px;
+	margin-bottom:10px;
+	width: 24px;
+	height: 24px;
+	display:flex;
+	gap: 5px;
+	padding-bottom:20px;
+	color:#f7f7f7;
+}
+.fa-brands {
+	font-size:30px;
+}
+/* MEDIA SECTION*/
+
+@media screen and (max-width: 400px) {
+	.hello-section {
+		padding:0;
+	}
+	.hello-section h1 {
+		font-size: 3.5rem;
+	}
+	.hello-section h2 {
+		font-size: 2rem;
+	}
+}
+
+@media screen and (max-width: 600px) {
+	
+	.card-img-text img {
+		width:100%;
+		object-fit:cover;
+		flex-wrap:column;
+	}
+	.card-img-text {
+		flex-direction: column;
+	}
+	.hello-section {
+		padding-left:0;
+	}
+}
+@media screen and (max-width: 800px) {
+	.card-img-text {
+		flex-direction: column;
+	}
+	.card-img-text img {
+		width: 100%;
+		object-fit: cover;
+		margin-left:0;
+	}
```

Comment:
a lot of these styles repeat between the two media queries, which means that you probably can get rid of half of them